app/helpers/underscore-text.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Helper } = Ember;
+
+export function underscoreText(params) {
```

Comment:
Please remove this helper and use the `underscore` helper from [ember-cli-string-helpers](https://github.com/romulomachado/ember-cli-string-helpers) which is already a part of this project.